### PR TITLE
Debian 10 sbom generation changes

### PR DESCRIPTION
**Pull request by koln67**
Add optional functionality to generate the SBOM for debian 10 images, if run_sbom_bool is set to true.

### CODE REVIEW

File: daisy_workflows/image_build/debian/build_fai.py [hunk 1]
```diff
@@ -49,6 +50,53 @@ def mycopytree(src, dst):
...
+    raise Exception('Could not determine the sector size')
+
+  partitions = ptable.get('partitions', None)
+  if partitions is None:
+    raise Exception('No partitions found in partition table')
+
+  for partition in partitions:
+    partition_type = partition['type']
+    # Get the offset for mounting the disk
+    if partition_type == Linux_Filesystem_GUID:
```
Comment (dorileo) [inline daisy_workflows/image_build/debian/build_fai.py:75]: you are not using partition_type anywhere else, so you can make it shorter:
```python
if partition['type'] == Linux_Filesystem_GUI:
...
```

File: daisy_workflows/image_build/debian/build_fai.py [hunk 2]
```diff
@@ -49,6 +50,53 @@ def mycopytree(src, dst):
     return dst
 
 
+# get number of bytes to offset when mounting the raw disk image file
```
Comment (hopkiw) [inline daisy_workflows/image_build/debian/build_fai.py:53]: we should not be modifying the build steps for debian; sbom generation should be its own separate process

**Pull request closed without merging**